projects/storefrontlib/src/cms-components/asm/asm-main-ui/asm-main-ui.component.ts

```diff
@@ -55,6 +61,11 @@ export class AsmMainUiComponent implements OnInit {
         }
       })
     );
+    this.subscription.add(
```

Comment:
Can we put this in an async pipe instead?

Reply:
fixed